Apps/AdminWebClient/src/ClientApp/src/services/restCsvExportService.ts

```diff
@@ -4,16 +4,13 @@ import { ICsvExportService } from "@/services/interfaces";
 @injectable()
 export class RestCsvExportService implements ICsvExportService {
     private readonly BASE_URI: string = "v1/api/CsvExport";
-    public downloadUserInfoCSV(): void {
-        this.downloadCsvExport("GetUserProfiles");
+    public getUserProfilesExportUrl(): string {
+        return `${this.BASE_URI}/GetUserProfiles`;
```

Comment:
We can use the configuration service to pass the address. So this service is not needed anymore

Reply:
right, this service seems to be over-kill. I will remove it and add params to configuration service then.